docs/restoring-dump-from-live.md

```diff
@@ -0,0 +1,72 @@
+# Restoring a dump from prod/staging
+
+_Note_ The below assumes you have access to the dxw Dalmatian infrastructure,
+and have `dalmatian-tools` installed. If you don't have access, and need to,
+contact a member of the dxw ops team.
+
+## Creating the dump
+
+Access the container as described in [Console Access](./console-access.md) and
+run the following command:
+
+```bash
+./manage.py dumpdata --natural-foreign --indent 2 \
+    -e contenttypes -e auth.permission -e postgres_search.indexentry \
+    -e wagtailcore.groupcollectionpermission \
+    -e wagtailcore.grouppagepermission -e wagtailimages.rendition \
+    -e sessions > /tmp/$(date +%Y%m%d).json
+```
+
+## Copying the file from the live environment
+
+List the EC2 instances in your chosen environment (where `ENV` is one of `prod`
+or `staging`):
+
+```bash
+dalmatian ecs ec2-access -i nhsx-website -e ENV -l
+```
+
+Access the EC2 instance (where `INSTANCE_ID` is the ID of the instance you
+want to connect to):
+
+```bash
+dalmatian ecs ec2-access -i nhsx-website -e ENV -I INSTANCE_ID
+```
+
+Switch to Sudo:
+
+```bash
+sudo bash
+```
+
+List the containers to get the name of your container:
+
+```bash
+docker ps
+```
+
+Copy the file from the relevant container:
+
+```bash
+docker cp CONTAINER_NAME:/tmp/$(date +%Y%m%d).json /tmp/$(date +%Y%m%d).json
+```
+
+Upload the file to S3 (where `ENVIRONMENT` is one of `staging` or `prod`)
```

Comment:
I actually have a working file download command which I'm going to put in a PR for tomorrow.

Reply:
What's the status of this file download command, and where can I learn more?